src/includes/platforms/configuration/integrations/redux.mdx

```diff
@@ -0,0 +1,107 @@
+<PlatformContent includePath="configuration/redux-init" />
+
+If you have other enhancers or middleware such as `thunk`:
+
+```javascript
+const store = createStore(
+  rootReducer,
+  compose(applyMiddleware(thunk), sentryReduxEnhancer)
+);
+```
+
+<Alert level="info" title="Note">
+
+Sentry uses a redux **enhancer**. Pass the enhancer, as indicated above. Don't pass it to `applyMiddleware` and don't call the method when you pass it to the `createStore` method.
+
+</Alert>
+
+## Normalization Depth
+
+By default Sentry SDKs normalize any context to a depth of 3, which in the case of sending Redux state you probably will want to increase that. You do so by passing `normalizeDepth` to the `Sentry.init` call:
+
+```javascript
+Sentry.init({
+  dsn: "___DSN___",
+  normalizeDepth: 10, // Or however deep you want your state context to be.
+});
+```
+
+## Redux Enhancer Options
+
+Pass an options object as the first parameter to `Sentry.createReduxEnhancer` to configure it.
+
+<Alert level="warning" title="Note">
+
+We advise against sending sensitive information to Sentry, but if you do, we will try our best to filter out things such as user passwords.
+
+</Alert>
+
+`actionTransformer` (Function)
+
+: Used to remove sensitive information from actions. The first parameter passed to the function is the Redux action. Return `null` to not send the action to Sentry. By default we send all actions.
+
+```javascript
+const sentryReduxEnhancer = Sentry.createReduxEnhancer({
+  actionTransformer: (action) => {
+    if (action.type === "GOVERNMENT_SECRETS") {
+      // Return null to not log the action to Sentry
+      return null;
+    }
+    if (action.type === "SET_PASSWORD") {
+      // Return a transformed action to remove sensitive information
+      return {
+        ...action,
+        password: null,
+      };
+    }
+
+    return action;
+  },
+});
+```
+
+`stateTransformer` (Function)
+
+: Used to remove sensitive information from state. The first parameter passed to the function is the Redux state. Return `null` to not attach the state to events sent to Sentry. Note that if you choose not to send state to Sentry, your errors might not have the latest version of the state attached. By default, we attach all state changes.
+
```

Comment:
```suggestion
This can be used to remove sensitive information from state. The first parameter passed to the function is the Redux state. We attach all state changes by default. If you don't want to attach state changes to events being sent to Sentry, use `return null`. Note, that if you choose not to send state to Sentry, your errors might not have the latest version attached. 

```